cmd/flags.go

```diff
@@ -48,18 +49,22 @@ func getCommandList(cmd *cobra.Command) (commandList []string, err error) {
 	return commandList, nil
 }
 
-func getRegionList(cmd *cobra.Command) (regionList []string, err error) {
-	if regionList, err = cmdutil.GetFlagStringSlice(cmd, "region"); err != nil {
-		return nil, err
+func getRegionList(cmd *cobra.Command) ([]string, error) {
```

Comment:
What's the reason for removing the named return values? It keeps the `if foo; err != nil` statements tidy.

Reply:
Lack of leveraging the initialized values and the fact we were still using explicit return values. Since here all the variables are initialized in some form and the return statement style wasn't changed I didn't use them. Style preference too since this was what I first made when I worked through getting the regions. I can change it back if that is desired.